Snake/BusinessLayer/Engine.cs

```diff
@@ -6,6 +6,11 @@
 
 namespace BusinessLayer
 {
+    public enum direction
+    {
+        right, down, left, up
```

Comment:
Formatting-wise, you should follow the pattern elsewhere and keep each direction on its own line.